### ISSUE TITLE
Allow logging of the console

### ISSUE THREAD

**Issue by timstoop** [kubevirt/kubevirt#3718]
**Is this a BUG REPORT or FEATURE REQUEST?**:

/kind enhancement

**What happened**:

We have a situation where we would like to log the console. As we've been running KVM VMs for years directly on hardware, it's one of those debugging options that we're used to and really miss when switching to kubevirt. There are a lot of situations where problems happen (for instance kernel panics) that are very hard to debug without the console output.

Libvirt supports this logging quite easily in the config itself, for example like this:

```
    <console type='file'>
      <source path='/var/log/libvirt/qemu/serial/my-vm.log'/>
      <target type='virtio' port='1'/>
    </console>
```

It does not seem possible to set this up for VMs running via kubevirt. Although logging this to std would be an option, the nature of the output makes it very hard to parse for things like fluentd as for example kernel traces are just multiline events that will end up as multiple events in the log. But having the option to output this to a separate PVC or maybe even next to the disk image (assuming there's also some way to prevent the log from filling up the disk) would be very convenient and useful.

**Comment by rmohr** [comment]
Sounds indeed like a useful feature. We probably have to come up with a sane design, especially on where to store and how to retrieve the logs.

**Comment by nirmoy** [comment]
Hi  @rmohr,  I am wondering if we already have this implemented?
